lib/view/IsHetAlView.php

```diff
@@ -134,6 +134,12 @@ public function __construct(
 				$session->set('studeren', $tijd);
 				break;
 
+			case 'kring':
+				// Matcht 'kring 42', 'loremipsumkring', 'kringlezing', maar niet 'kringleidersinstructie'.
+				$vandaag = $agendaRepository->zoekRegexAgenda('/kring(?: \d+|\b|lezing\b)/i');
```

Comment:
Prettier zegt:
```suggestion
				$vandaag = $agendaRepository->zoekRegexAgenda(
					'/kring(?: \d+|\b|lezing\b)/i'
				);
```

Bij het runnen van prettier is `--plugin=@prettier/plugin-php` nodig, dit staat er niet standaard in, zal het fixen op master.